scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -0,0 +1,49 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'SOP'
+
+SOP-8-1EP_4.57x4.57mm_P1.27mm_EP4.57x4.45mm:
+  size_source: 'https://ww2.minicircuits.com/case_style/XX112.pdf'
+  body_size_x:
+    nominal: 4.57
+    tolerance: 0.01
+  body_size_y:
+    nominal: 4.57
+    tolerance: 0.01
+  overall_size_x:
+    nominal: 10.16
+    tolerance: 0.01
+  lead_width:
+    nominal: 0.38
+    tolerance: 0.01
+  lead_len:
+    nominal: 1.78
+    tolerance: 0.01
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 4
+
+  EP_size_x:
+    nominal: 4.57
+    tolerance: 0.01
+  EP_size_y:
+    nominal: 4.45
```

Comment:
Where does this come from? Parameter R is 4.57 in the datasheet. (Sadly there is no dimension given for the metal pad on the package it self. Only a suggested pad size.)

Reply:
I am assuming that this dimension comes from the fact that the EP looks like ending at the same line as the pads. Meaning 3*1.27+0.64 = 4.45. Could you add a note in the yaml file about that?
(Something like 

``` yaml
nominal: 4.45     # not directly given in datasheet. EP edge coincides with pads. (3*1.27+0.64 = 4.45)
```

Reply:
ah, now I understand what you were pointing at exactly. Yes, that's there I got this number from. I also added your comment to make it clear.